<launch>
<include file="$(dirname)/flatland_simulation.launch" />
<!----> RLvehicle_controller -->
<arg name="sensor_topic" default="vehicle/lidar_measurements"/>
<arg name="actuator_topic" default="vehicle/actuator_commands"/>

<param name="vehicle/actuator_topic" value="$(arg actuator_topic)" type="str"/>
<param name="vehicle/sensor_topic" value="$(arg sensor_topic)" type="str"/>

<node name="RL_vehicle_controller" pkg="racing" type="RL_controller.py" output="screen"/>
<!-- vehicle_timer (lap time measurement) -->
<arg name="position_topic" default="odometry/ground_truth"/>
<param name="vehicle/position_topic" value="$(arg position_topic)" type="str"/>
<node name="vehicle_timer" pkg="racing" type="vehicle_timer_node" output="screen"/>

</launch>